.pre-commit-config.yaml

```diff
@@ -29,6 +29,7 @@ repos:
           - "--max-complexity=18"
           - "--select=B,C,E,F,W,T4,B9"
           - "--ignore=E203,E266,E501,W503,F401,E402"
+          - "--exclude=docs"
```

Comment:
do you really need this lines if you specify files: ^src/pykiso

Reply:
It was in the setup.cfg, so I just migrated the contents 1 to 1

Reply:
strange the setup.cfg was only there for the pytest configuration